temporalcli/commands.workflow_trace.go

```diff
@@ -0,0 +1,67 @@
+package temporalcli
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"github.com/temporalio/cli/temporalcli/internal/trace"
+	"go.temporal.io/sdk/client"
+)
+
+func (c *TemporalWorkflowTraceCommand) run(cctx *CommandContext, _ []string) error {
```

Comment:
Can you have a high-level test or two that invokes the command using the harness (can see other commands for how this is done)

Reply:
Bump, I do think we want at least one test confirming the call from the CLI POV works

Reply:
They're great, thanks!